src/components/Shared/DeleteConfirmation.js

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useState } from 'react'
+import CustomTextField from '../Inputs/CustomTextField'
+import WindowToolbar from './WindowToolbar'
+
+const DeleteConfirmation = ({ window, props, obj }) => {
+  const [deleteConfirmation, setDeleteConfirmation] = useState('')
```

Comment:
having a formik instance is much better to keep the same methodology in our project instead of having 2 states to save the text typed in the field and to set the button disabled or no.
let's discuss this in our next call before you change it.